src/utils/database.ts

```diff
@@ -3994,6 +4056,33 @@ const updateMeetingType = async (
   return data?.[0] as MeetingType
 }
 
+const updateAvailabilityBlockMeetingTypes = async (
+  account_address: string,
+  availability_block_id: string,
+  meeting_type_ids: string[]
+) => {
+  // First verify the availability block exists and belongs to the account
+  const { data: block, error: blockError } = await db.supabase
+    .from('availabilities')
+    .select('id')
+    .eq('id', availability_block_id)
+    .eq('account_owner_address', account_address)
+    .single()
+
+  if (blockError || !block) {
+    throw new AvailabilityBlockNotFoundError()
+  }
+
+  await updateAssociations(
```

Comment:
Next time let's be more specific with the name I.e this should be updateMeetingTypeAssociations